spec/features/recipe_show_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe 'recipe#show', type: :feature do
+  before(:each) do
+    @user1 = User.create(
+      id: 3,
+      name: 'Munish',
+      email: 'munish@yahoo.com',
+      password: 'munish123',
+      confirmed_at: Time.now
+    )
+
+    sign_in @user1
+
+    @recipe1 = Recipe.create(
+      id: 1,
+      user_id: @user1.id,
+      name: 'sauce',
+      preparation_time: 100,
+      cooking_time: 30,
+      description: 'test',
+      public: true
+    )
+
+    visit user_recipe_url(id: @recipe1.id, user_id: @recipe1.user.id)
+  end
+  describe '#Indexpage' do
+    it 'can view recipe preparation_time.' do
+      expect(page).to have_content(@recipe1.preparation_time)
+    end
+
+    it 'can view recipe description.' do
+      expect(page).to have_content(@recipe1.cooking_time)
+    end
+
```

Comment:
Good job implementing some of the project requirements but I noticed you have some failing tests. It would be nice to fix this issue.
![Screenshot 2023-10-13 at 10 46 21 PM](https://github.com/hmunish/ror-recipe-app/assets/79506105/43c4cf56-c331-44f2-bd60-7d1ba245405f)

Reply:
@KayLemba, thank you so much sir for the assistance you gave. It means a lot. I will fix the bugs & submit for code review again. Thank you so much again.